provisioning/standalone/docs/example.md

```diff
@@ -0,0 +1,94 @@
+# Example
+
+In this example, we are going to learn how to deploy a simple echo server using **golang**, which will answer to a **GET HTTP request** with a predefined message.
+
+### Create the server
+
+Create a **main.go** file which will contain the source code of the server and **copy/paste** the following code inside **main.go**
+
+```go
+import (
+	"fmt"
+	"net/http"
+	"os"
+)
+
+func hello(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Hello!\n")
+}
+
+func main() {
+	port := os.Args[1]
+	basepath := os.Args[2]
+	http.HandleFunc(fmt.Sprintf("%s/", basepath), hello)
+	err := http.ListenAndServe(fmt.Sprintf("0.0.0.0:%s", port), nil)
+	if err != nil {
+		fmt.Println(err)
+	}
+}
+```
+
+Let's test it by running it with `go run 8888 /hello`.
+
+In the meanwhile open another terminal and launch the command `curl localhost:8888/hello`, and you will get `Hello!` as response.
+
+### Build the container
+
+Let's create a **Dockerfile** ` and **copy/paste** the following code inside of it.
+
+```dockerfile
+FROM golang:1.18
+ENV CROWNLABS_LISTEN_PORT=8888
+ENV CROWNLABS_BASE_PATH=/basepath
+COPY main.go main.go
+RUN go build main.go
+RUN useradd -ms /bin/bash myuser
+USER myuser
+CMD exec ./main ${CROWNLABS_LISTEN_PORT} ${CROWNLABS_BASE_PATH}
```

Comment:
To achieve smaller images (avoiding multi-stage builds), you could build the executable in the local machine and then copy only that in the docker. That is what many projects do after all.